core/src/main/java/fr/sncf/osrd/api/InfraCacheStatusEndpoint.java

```diff
@@ -44,8 +44,8 @@ public Response act(Request req) throws IOException {
         // Parse request input
         try {
             var body = new RqPrint(req).printBody();
-
-            if (body.equals("")) {
+            System.out.println(body);
+            if (body.equals("")|| body.equals("{\"infra\":null}")) {
```

Comment:
I feel like testing equality of the body against a string isn't very pretty, maybe you could do something around
```suggestion
String infra_id;
if (!body.equals("")) {
    infra_id = adapterRequest.fromJson(body).infra
}
if (infra_id == null)) {
```